plugins/com.google.cloud.tools.eclipse.appengine.newproject/plugin.properties

```diff
@@ -2,4 +2,7 @@ google = Google
 Bundle-Vendor = Google Inc.
 Bundle-Name = Wizards for Google Cloud Platform Apps
 
+gcp.category = Google Cloud Platform
+
+wizard.name = Google App Engine Standard Java Project
```

Comment:
Tangential note: is there a quick way to read these properties from the code? If so, I should do that in https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/pull/156